src/NLog/LayoutRenderers/MachineNameLayoutRenderer.cs

```diff
@@ -48,7 +48,7 @@ namespace NLog.LayoutRenderers
     [AppDomainFixedOutput]
     [ThreadAgnostic]
     [ThreadSafe]
-    public class MachineNameLayoutRenderer : LayoutRenderer, IStringValueRenderer
```

Comment:
this is this removed? I don't understand in the context of this PR

Reply:
There is no need for IStringValueRenderer when having the attribute `[AppDomainFixedOutput]`

Reply:
Have added unit-tests to verify compatibility between internal interfaces and attributes.

Reply:
great!